src/services/analytics/events/wallet.ts

```diff
@@ -35,4 +35,9 @@ export const WALLET_EVENTS = {
     action: 'Confirm message',
     category: WALLET_CATEGORY,
   },
+  RELAYED_EXECUTION: {
+    event: EventType.META,
+    action: 'Relayed execution',
+    category: WALLET_CATEGORY,
+  },
```

Comment:
Technically the `TxEvent.RELAYING` event is dispatched before it was successfully executed. Should we rename this to `Relaying execution` accordingly?

Reply:
It's not a verb in past tense, it's supposed to mean "An execution that's being relayed"